src/mcts/search.cc

```diff
@@ -502,7 +519,7 @@ EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
     }
     if (edge.GetN() + offset > max_n) {
       max_n = edge.GetN() + offset;
-      max_eval = edge.GetQ(fpu);
+      max_eval = edge.GetQ(fpu, is_odd_depth);
```

Comment:
What is the intention here (and two more places later on)?

Reply:
Not sure I understand what you ask.
We are limiting low-Q moves from being picked due to temperature. As Q incorporates draw score during visits gathering, it also makes sense to do that during application of the temperature.
`is_odd_depth = false` is because this function is currently only called for the root node.

Reply:
I meant that (unless I'm seriously confused) the second argument to GetQ is `draw_score`,so why pass `is_odd_depth`?

Reply:
Good catch! Fixed.